app/src/utils/utils.test.tsx

```
@@ -50,25 +48,24 @@ describe("apiPOSTCall()", () => {
           "https://api.repo.nypl.org/api/v2/items/local_image_id/105180",
           { uuid: ["uuid1", "uuid2"] }
         )
-      ).toBe("success");
+      ).toStrictEqual({ nyplAPI: { response: "success" } });
     });
   });
 });
 
 describe("getItemsCountFromUUIDs()", () => {
-  it("should return an empty object with a bad request", async () => {
+  it("should return an error with a bad request", async () => {
```

nit: "throw" an error rather than "return"